### PR TITLE
adds a bundle config yaml example

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

### Description
adding another example of yaml config and bundle parsing.

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

File: bundle/introducing_config/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+```yaml
+imports:
+  - $import glob
+  - $import matplotlib.pyplot as plt
+```
+
+These two statements will be evaluated and the imported modules will be available throughout the config file.
+By default, the parser imports `monai`'s modules and `import numpy as np` for you, so you don't need to add them in the config file.
+
+### Running the `display` component
```
Comment (KumoLiu) [inline bundle/introducing_config/README.md:38]: Hi @wyli, maybe we can put this one after L110 to make it more consistent with the equivalent Python code.

File: bundle/introducing_config/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+plt.imshow(first_pair["f_img"][0], cmap="gray")
+plt.subplot(1, 2, 2)
+plt.imshow(first_pair["m_img"][0], cmap="gray")
+plt.show()
+
+```
+
+## Topics not covered but possible in the config
+
+- Running customized Python components (made available on the `PYTHONPATH`).
```
Comment (KumoLiu) [inline bundle/introducing_config/README.md:154]: Maybe we can put a link here for how to use the customized components. Just show an example I find in MONAI model zoo.

https://github.com/Project-MONAI/model-zoo/blob/987f49fe9e0f20f047a2252c1ace38bd760cfb33/models/lung_nodule_ct_detection/configs/train.json#L95-L100

File: bundle/introducing_config/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+- Overriding the component in `example.yaml` using, for example, `--id=new_value` in the command line.
+- Multiple configuration files and cross-file references.
```
Comment (KumoLiu) [inline bundle/introducing_config/README.md:156]: Also for this one we can find an example in the model zoo, maybe we can show the link?
https://github.com/Project-MONAI/model-zoo/tree/dev/models/spleen_ct_segmentation

**Comment by Nic-Ma** [comment]
Hi @wyli ,

May I know what's the feature request for this PR? Why add another example?

Thankss.

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> May I know what's the feature request for this PR? Why add another example?
> 
> Thankss.

I got questions about how to use the config, there's no yaml example although the algo templates are based on yaml.

**Comment by Nic-Ma** [comment]
> > Hi @wyli ,
> > May I know what's the feature request for this PR? Why add another example?
> > Thankss.
> 
> I got questions about how to use the config, there's no yaml example although the algo templates are based on yaml.

Hi @wyli ,

This example is a YAML config:
https://github.com/Project-MONAI/tutorials/blob/main/bundle/hybrid_programming/configs/net_inferer.yaml
Or maybe you can change some config of the spleen bundle example to YAML?
https://github.com/Project-MONAI/tutorials/tree/main/bundle/spleen_segmentation/configs

Thanks.

**Comment by wyli** [comment]
> > > Hi @wyli ,
> > > May I know what's the feature request for this PR? Why add another example?
> > > Thankss.
> > 
> > 
> > I got questions about how to use the config, there's no yaml example although the algo templates are based on yaml.
> 
> Hi @wyli ,
> 
> This example is a YAML config: https://github.com/Project-MONAI/tutorials/blob/main/bundle/hybrid_programming/configs/net_inferer.yaml Or maybe you can change some config of the spleen bundle example to YAML? https://github.com/Project-MONAI/tutorials/tree/main/bundle/spleen_segmentation/configs
> 
> Thanks.

this is an example compares the python code and the config, I don't think existing ones explain the details.

**Pull request merged**